client/landing/jetpack-cloud/components/missing-credentials/index.tsx

```diff
@@ -0,0 +1,45 @@
+/**
```

Comment:
Lets add a readme to this component. So that we better understand how it is supposed to be used. :)